specs/pvs/corla.pvs

```diff
@@ -254,6 +271,9 @@ BEGIN
   full_hand_count_contest: TYPE FROM contest
 % @todo Contest name uniqueness.
 
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   outcome: TYPE
```

Comment:
I'll update the preface wrt the glossary.  As mentioned on Slack, @mkiniry is double-checking the relation between the glossary terms and the specification.  We talked over the notion of nouns, adjectives/adverbs, and verbs being types, predicate types, and functions, and the means by which that we will update the specification with @description and @explanation annotations so that the glossary becomes an automatically generated document.

I'm not going to dive into those crazy classes of contest_outcomes for the moment.  Let's just add a @note annotation that reminds us to return to that matter when necessary.

Note that `contest_outcome` is currently under-specified so as to leave open the possibility of encoding these subtleties as necessary for this release.  E.g., a legitimate encoding right now could simply be something like `[# winner: option #]`.  I didn't intend to refine this until we go over the Dominion artifacts.